### PR TITLE
Masks code-style massage

### PR DESCRIPTION
**Pull request by lucasart**
No functional change.

BENCH : 7,840,295

### CODE REVIEW

**Comment by AndyGrant** [comment]
Lets change ``initializeMasks()`` to just ``initMasks()``. You did that with the attacks, and I just rewrote some of the TT stuff and used ``initTT()``

Good cleanup on the pawn connected masks. I think I put that there before I had ``pawnAttacks`` for a square and a colour defined somewhere accessible.

File: src/masks.c [hunk 1]
```diff
@@ -34,78 +34,64 @@ uint64_t PawnConnectedMasks[COLOUR_NB][SQUARE_NB];
...
-        OutpostSquareMasks[BLACK][i] = PassedPawnMasks[BLACK][i] & ~Files[fileOf(i)];
+    for (int s = 0; s < SQUARE_NB; s++) {
+        const uint64_t files = IsolatedPawnMasks[s] | Files[fileOf(s)];
+
+        PassedPawnMasks[WHITE][s] = files;
+        for (int r = rankOf(s); r >= 0; r--)
+            PassedPawnMasks[WHITE][s] &= ~Ranks[r];
+
+        PassedPawnMasks[BLACK][s] = files;
+        for (int r = rankOf(s); r <= 7; r++)
```
Comment (AndyGrant) [inline src/masks.c]: `` r < RANK_NB`` Only other thing. Missed it at first.

**Comment by lucasart** [comment]
Do you prefer to use arrays directly for masks? Or to hide them from the API and only expose accessor functions that perform assert bounds checking ? The latter adds a few lines of code, but it's trivial copy pasta code (assert), and it helps catch bugs.

**Comment by AndyGrant** [comment]
I would be happy with functions housed in ``masks.c``, makes more sense that way -- and error checking is trivial.

**Comment by lucasart** [comment]
updated. please check.

**Comment by AndyGrant** [comment]
Did you mean to have each function declared with ``extern`` in ``masks.h``?

**Comment by lucasart** [comment]
updated

**Pull request merged**